js/sia.js

```diff
@@ -140,22 +141,39 @@ function SiadWrapper () {
       return false
     }
 
-    // Set siad folder as configured siadPath
+    // Set siad folder as configured siad.path
     var processOptions = {
       cwd: siad.path
     }
+
+    // If the detached option is set, spawn siad as a separate process to be
+    // run in the background after the parent process has closed
+    if (siad.detached) {
+      let log = require('path').join(siad.path, 'out.log')
+      let out = fs.openSync(log, 'a')
+      let err = fs.openSync(log, 'a')
+      processOptions.detached = true
+      processOptions.stdio = [ 'ignore', out, err ]
+    }
+
+    // Spawn siad
     const Process = require('child_process').spawn
     var daemonProcess = new Process(siad.fileName, processOptions)
 
-    // Listen for siad erroring
-    // TODO: Attach these to siad if it's already running
-    daemonProcess.on('error', function (err) {
-      self.emit('error', err)
-    })
-    daemonProcess.on('exit', function (code) {
-      siad.running = false
-      self.emit('exit', code)
-    })
+    // Exclude it from the parent process' event loop if detached
+    if (siad.detached) {
+      daemonProcess.unref()
+    } else {
+      // Listen for siad erroring
+      // TODO: Attach these to siad if it's already running
```

Comment:
If siad is running as a background process, `sia.js` won't be able to listen to its events and can only interact with it through the API. It's a minor annoyance.